regexes/oss.yml

```diff
@@ -555,7 +569,7 @@
   name: 'CentOS Stream'
   version: '$1'
 
-- regex: '.+.el(\d+(?:[_\.]\d+)*).(?:centos|x86_64)'
+- regex: '.+.el(\d+(?:[_\.]\d+)*)'
```

Comment:
we need a more strict rule. what about?
`Linux/(?:\d+\.[\d.-]+)\.el(\d+(?:[_.]\d+)*)`

Reply:
@sanchezzzhak I've reverted the commits for CentOS, since I'm not sure if this user agent is for Red Hat Enterprise Linux or CentOS

`python-requests/2.7.0 CPython/2.7.3 Linux/2.6.18-308.el5`